tests/oss-fuzz/build.sh

```diff
@@ -75,6 +75,8 @@ if [ "$FUZZING_ENGINE" == "libfuzzer" ]
 then
   export CXXFLAGS="${CXXFLAGS} -DFUZZTEST_COMPATIBILITY_MODE"
   export EXTRA_CMAKE_FLAGS="-DAVIF_ENABLE_FUZZTEST=ON -DFUZZTEST_COMPATIBILITY_MODE=libfuzzer"
+else
+  export EXTRA_CMAKE_FLAGS=""
```

Comment:
No need to revert `CXXFLAGS`?

Reply:
nope, it is already given by the fuzzer (with all the -fsanitize ...)

Reply:
I don't think you need to export EXTRA_CMAKE_FLAGS; no subprocess depends on the value. You can just assign it to something.

If CXXFLAGS is already exported, assigning to it will update the exported value as well. Though it doesn't hurt to be explicit.

Reply:
Fixed in https://github.com/AOMediaCodec/libavif/pull/2044